### PR TITLE
Add cloudPcSupportedRegion resource topic

### PR DESCRIPTION
**Pull request by RuiHou105**

### CODE REVIEW

**Comment by jasonjoh** [comment]
Docs Build status updates of commit _[af168d6](https://github.com/microsoftgraph/microsoft-graph-docs/commits/af168d6ac46cd18baf82a7466384beea0bc3f82c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/virtualendpoint-list-supportedregions.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/rui/AddSupportedRegions/api-reference/beta/api/virtualendpoint-list-supportedregions.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/virtualendpoint-list-supportedregions?view=graph-rest-beta&branch=pr-en-us-14111) (graph-rest-beta) |
[api-reference/beta/resources/cloudpcsupportedregion.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/rui/AddSupportedRegions/api-reference/beta/resources/cloudpcsupportedregion.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/cloudpcsupportedregion?view=graph-rest-beta&branch=pr-en-us-14111) (graph-rest-beta) |
[api-reference/beta/resources/virtualendpoint.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/rui/AddSupportedRegions/api-reference/beta/resources/virtualendpoint.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/virtualendpoint?view=graph-rest-beta&branch=pr-en-us-14111) (graph-rest-beta) |
[api-reference/beta/toc.yml](https://github.com/microsoftgraph/microsoft-graph-docs/blob/rui/AddSupportedRegions/api-reference/beta/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/virtualendpoint-list-supportedregions?view=graph-rest-beta&branch=pr-en-us-14111) (graph-rest-beta) |
[changelog/Microsoft.CloudManagedDesktop.json](https://github.com/microsoftgraph/microsoft-graph-docs/blob/rui/AddSupportedRegions/changelog/Microsoft.CloudManagedDesktop.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C9%5C2%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202109020727083936-14111%5Cworkflow_report.html?sv=2018-03-28&sr=b&sig=e8sx%2BANzrDLfqJrEi6JiGf3xriWZ6dfxzSzibox0mWM%3D&st=2021-09-02T07%3A24%3A31Z&se=2021-10-03T07%3A29%3A31Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by RuiHou105** [comment]
Roadmap workitem:
Feature 11024: [Devices and apps][Cloud PC] Add cloudPcSupportedRegion entity

File: api-reference/beta/api/virtualendpoint-list-supportedregions.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+doc_type: apiPageType
+---
+
+# List supportedRegions
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+List properties and relationships of the [cloudPcSupportedRegion](../resources/cloudpcsupportedregion.md) objects.
```
Comment (isvargasmsft) [inline api-reference/beta/api/virtualendpoint-list-supportedregions.md:16]: Lines 3 and 16: This line is used by search engines to display results. It is a good practice to write these descriptions in the way we would search for this topic in Bing. 
For **example**:
- List the regions available to create cloud PC connections to the specified virtualEndPoint.

File: api-reference/beta/api/virtualendpoint-list-supportedregions.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+## HTTP request
+
+<!-- {
+  "blockType": "ignored"
+}
+-->
+
+``` http
+GET /deviceManagement/virtualEndpoint/supportedRegions
```
Comment (isvargasmsft) [inline api-reference/beta/api/virtualendpoint-list-supportedregions.md:38]: Line 38: Shouldn't we specify a particular virtualEndpoint-ID?
/deviceManagement/virtualEndpoint/ID/supportedRegions ?

File: api-reference/beta/api/virtualendpoint-list-supportedregions.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+## Request body
+
+Do not supply a request body for this method.
+
+## Response
+
+If successful, this method returns a `200 OK` response code and a collection of [cloudPcSupportedRegion](../resources/cloudpcsupportedregion.md) objects in the response body.
+
+## Examples
```
Comment (isvargasmsft) [inline api-reference/beta/api/virtualendpoint-list-supportedregions.md:59]: Please change to singular "Example".

File: api-reference/beta/resources/cloudpcsupportedregion.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+doc_type: resourcePageType
+---
+
+# cloudPcSupportedRegion resource type
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Represent a supported region in which on-premises network connection can be established for Cloud PCs.
```
Comment (isvargasmsft) [inline api-reference/beta/resources/cloudpcsupportedregion.md:16]: Lines 3 and 16: let's rephrase for simplicity and to active voice. For example:
Represents a region where Cloud PCs can connect to an on-premises network.
or
Represents a region where Cloud PCs can establish an on-premises network connection. 
or
Represents a region supported to establish on premises network connections for Cloud PCs.

File: api-reference/beta/resources/cloudpcsupportedregion.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+[!INCLUDE [cloudpc-api-preview](../../includes/cloudpc-api-preview.md)]
+
+## Methods
+
+|Method|Return type|Description|
+|:---|:---|:---|
+|[List cloudPcSupportedRegions](../api/virtualendpoint-list-cloudpcs.md)|[cloudPcSupportedRegion](../resources/cloudpcsupportedregion.md) collection|List [cloudPcSupportedRegion](../resources/cloudpcsupportedregion.md) objects and their properties.|
```
Comment (isvargasmsft) [inline api-reference/beta/resources/cloudpcsupportedregion.md:24]: Please make the description in this line match the description in the List topic after the comments are implemented.

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Excellent work in this PR @RuiHou105. Thank you very much!

Just a few suggestions for phrasing and I'd like to clarify the request syntax. 

Please feel free to ping me on the side if you have any questions. Here to help!

Isaac Vargas

File: api-reference/beta/api/virtualendpoint-list-supportedregions.md (hunk 2 above)
Comment (RuiHou105) [reply api-reference/beta/api/virtualendpoint-list-supportedregions.md:38]: The supportedRegions is one of virtualEndpoint's navigation properties, so virtualEndpoint is used to route to supportedRegions which is a collection of entity cloudPcSupportedRegion. Thus, we don't need ID here.

File: api-reference/beta/api/virtualendpoint-list-supportedregions.md (hunk 1 above)
Comment (RuiHou105) [reply api-reference/beta/api/virtualendpoint-list-supportedregions.md:16]: Thanks for your suggestion. But one thing I want to clarify is that there is no "specified" virtualEndpoint ( you can see my another reply in your comment below). So, no need to add ID in the URL and no need to add "specified". Are you good with this one - List the supported regions available to create Cloud PC connections?

File: api-reference/beta/api/virtualendpoint-list-supportedregions.md (hunk 3 above)
Comment (RuiHou105) [reply api-reference/beta/api/virtualendpoint-list-supportedregions.md:59]: Updated. Thank you.

File: api-reference/beta/resources/cloudpcsupportedregion.md (hunk 4 above)
Comment (RuiHou105) [reply api-reference/beta/resources/cloudpcsupportedregion.md:16]: Updated to version3.

File: api-reference/beta/api/virtualendpoint-list-supportedregions.md (hunk 1 above)
Comment (isvargasmsft) [reply api-reference/beta/api/virtualendpoint-list-supportedregions.md:16]: Thank you for the clarification. And yes: List the supported regions available to create Cloud PC connections
Thank you very much!

**Review by AshleyYangSZ — APPROVED** [review]

File: api-reference/beta/api/virtualendpoint-list-supportedregions.md (hunk 1 above)
Comment (RuiHou105) [reply api-reference/beta/api/virtualendpoint-list-supportedregions.md:16]: Updated. Please check.

File: api-reference/beta/resources/cloudpcsupportedregion.md (hunk 5 above)
Comment (RuiHou105) [reply api-reference/beta/resources/cloudpcsupportedregion.md:24]: Updated description. Please check.

**Comment by RuiHou105** [comment]
@isvargasmsft All comments resolved. Could you please take a second look at this PR? Thanks!

**Review by isvargasmsft — APPROVED** [review]
Thank you very much for your work in this PR!

**Review by Lauragra — APPROVED** [review]

**Pull request merged**